.travis.yml

```diff
@@ -17,6 +17,9 @@ jobs:
     - stage: platform-test
       node_js: "node"
       os: osx
+    - stage: platform-test
+      node_js: "11"
+      os: linux
     - stage: platform-test
       node_js: "10"
```

Comment:
Could probably flip the 10s over to `lts/dubnium` now. Not sure if that makes it clearer or not, but I was just seeing that we/I did that for the other LTS in the matrix

Reply:
I'm finding the lts labels confusing now. I've switched them all to be version numbers instead.